.github/workflows/publish_container.yml

```diff
@@ -0,0 +1,54 @@
+name: Publish Container Image
+
+on:
+  workflow_dispatch:
+    inputs:
+      image_version:
+        description: "Image version"
+        default: "dev"
+        type: string
+        required: False
+  workflow_call:
+    inputs:
+      image_version:
+        description: "Image version"
+        default: "dev"
+        type: string
+        required: False
+
+jobs:
+  publish_container_image:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
```

Comment:
In the name of consistency and clarity maybe it will be better to add:
`- name: Checkout release tag` as it was in the `cd.yml` workflow.